oidc-api/src/main/java/uk/gov/di/authentication/oidc/validators/RequestObjectAuthorizeValidator.java

```diff
@@ -214,10 +214,28 @@ private static boolean isSignatureValid(SignedJWT signedJWT, String publicKey) {
     private Optional<AuthRequestError> validateVtr(JWTClaimsSet jwtClaimsSet, URI redirectURI) {
         List<String> authRequestVtr = new ArrayList<>();
         try {
-            authRequestVtr =
-                    Objects.isNull(jwtClaimsSet.getClaim(VTR_PARAM))
-                            ? emptyList()
-                            : List.of(jwtClaimsSet.getClaim(VTR_PARAM).toString());
+            if (Objects.nonNull(jwtClaimsSet.getClaim("vtr"))) {
+                if (jwtClaimsSet.getClaim("vtr") instanceof String vtr) {
+                    authRequestVtr = List.of(vtr);
+                } else if (jwtClaimsSet.getClaim("vtr") instanceof List<?> vtrList
+                        && vtrList.stream().allMatch(vtr -> vtr instanceof String)) {
+                    authRequestVtr =
+                            List.of(
+                                    String.format(
+                                            "[\"%s\"]",
```

Comment:
This formatting feels a bit odd - is it required to satisfy `VectorOfTrust.parseFromAuthRequestAttribute()`? 
If so would it be better to create a new parsing method in `VectorOfTrust`?

Reply:
Yeah it's required to get it into the format to keep it consistent with the query parameter approach. I didn't realise how ugly it looked after spotless had its way with it - although I'd argue that the parsing method should maybe be in `RequestObjectToAuthRequestHelper`, as it's due to the formatting of the `RequestObject` that we have to do this

Reply:
Can we use objectMapper.writeAsString or similar to just serialise this?